src/ralph/admin/templatetags/ralph_tags.py

```diff
@@ -14,6 +14,10 @@
 
 register = Library()
 
+SENTRY_ENABLED = getattr(settings, 'SENTRY_ENABLED', False)
```

Comment:
please add default values to settings for this variables and use env. variables to set value in settings